kedro-telemetry/README.md

```diff
@@ -20,42 +20,39 @@ for further information on the intent of the data collection and what data is co
+2. **CLI Option When Creating a New Project**:
+   When creating a new project, you can use the command:
 
-When you next run the Kedro CLI you will be asked for consent to share usage analytics data for the purposes explained in the privacy notice, and a `.telemetry` YAML file will be created in the project root directory. The variable `consent` will be set according to your choice in the file, e.g. if you consent:
+   ```console
+   kedro new --telemetry=yes/no
+   ```
+   This will create a `.telemetry` file inside the new project's folder with `consent: true/false` accordingly. This file will be used when executing Kedro commands within that project folder. Note that telemetry data about the execution of the `kedro new` command will still be sent if telemetry has not been disabled using environment variables.
 
-```yaml
-consent: true
-```
+   >*Note:* The `.telemetry` file should not be committed to `git` or packaged in deployment. In `kedro>=0.17.4` the file is git-ignored.
 
->*Note:* The `.telemetry` file should not be committed to `git` or packaged in deployment. In `kedro>=0.17.4` the file is git-ignored.
-
-## How do I withdraw consent?
+3. **Modify or Create the `.telemetry` File**:
+   If the `.telemetry` file exists in the root folder of your Kedro project, set the `consent` variable to `false`. If the file does not exist, create it with the following content:
+     ```yaml
+     consent: false
+     ```
 
-To withdraw consent, you can change the `consent` variable to `false` in `.telemetry` YAML by editing the file in the following way:
+4. **Uninstall the Plugin**:
+   Remove the `kedro-telemetry` plugin:
 
-```yaml
-consent: false
-```
-You can also set `DO_NOT_TRACK` or `KEDRO_DISABLE_TELEMETRY` environment variable to `True`.
-
-Or you can uninstall the plugin:
-
-```console
-pip uninstall kedro-telemetry
-```
+   ```console
+   pip uninstall kedro-telemetry
+   ```
 
 ## What happens when I withdraw consent?
 
-Data will only be collected if consent is given. Otherwise, if consent was explicitly denied or withdrawn, the message below will be printed out on every Kedro CLI invocation. If you explicitly deny consent from the beginning, no data will be collected. If you withdraw consent later, the processing of data will be stopped from that moment on.
+Data collection for telemetry is enabled by default. If consent was explicitly denied or withdrawn, the message below will be printed out on every Kedro CLI invocation. If you explicitly deny consent from the beginning, no data will be collected. If you withdraw consent later, the processing of data will be stopped from that moment on.
```

Comment:
> the message below will be printed out on every Kedro CLI invocation

This is outside the scope of this PR, but should we print the message only once at the first invocation? Otherwise, it might be annoying for the user.

Relates to: https://github.com/kedro-org/kedro-plugins/issues/781

Reply:
I tried to get a clear answer about whether we should print something when telemetry is disabled in the issue: [#727](https://github.com/kedro-org/kedro-plugins/issues/727). However, I haven't received a response yet, so I left it as it is. I think we can create a new issue to discuss this further.

Reply:
I agree it can be annoying to see that every time. But it feels like too much work to print it "just the first time" (per project? per user?). What about not printing anything if telemetry is disabled?

Reply:
That makes sense. I agree that it's better to remove the notification in this case. If a person has disabled telemetry, they are likely aware that it is no longer active. Additionally, we will stop sending successful telemetry notifications to alleviate any concerns that telemetry is still being sent.

Reply:
Done in https://github.com/kedro-org/kedro-plugins/pull/792, docs updated accordingly in current PR.